lambdas/data-migration2/main.tf

```diff
@@ -89,10 +89,12 @@ resource "aws_lambda_function" "data_migration2" {
   environment {
     variables = {
       databaseCredentialSecretArn = var.rds_user_access_secret_arn
-      ExecutionsTable = var.dynamo_tables.executions.name
-      GranulesTable = var.dynamo_tables.granules.name
-      PdrsTable = var.dynamo_tables.pdrs.name
-      dbHeartBeat = var.rds_connection_heartbeat
+      ExecutionsTable             = var.dynamo_tables.executions.name
+      GranulesTable               = var.dynamo_tables.granules.name
+      PdrsTable                   = var.dynamo_tables.pdrs.name
+      dbHeartBeat                 = var.rds_connection_heartbeat
+      system_bucket               = var.system_bucket
```

Comment:
I'm wondering if we should use a more generic configuration variable name here as the output doesn't *have* to go to the system bucket, so much as *a* bucket.